thirdparty/embree/common/sys/sysinfo.cpp

```diff
@@ -672,9 +672,9 @@ namespace embree
     static int nThreads = -1;
     if (nThreads != -1) return nThreads;
 
-#if defined(__MACOSX__) || defined(__ANDROID__)
-    nThreads = sysconf(_SC_NPROCESSORS_ONLN); // does not work in Linux LXC container
-    assert(nThreads);
+#if defined(__MACOSX__) || defined(__ANDROID__) || defined(__OPEN_HARMONY__)
```

Comment:
Changes in the `thirdparty` should also include `patch` file for the in the corresponding library `patches` (see existing patches).

New patches should be mentioned in the `thirdparty\readme.md`.